lib/std/rand.zig

```diff
@@ -28,6 +28,25 @@ pub const Sfc64 = @import("rand/Sfc64.zig");
 pub const RomuTrio = @import("rand/RomuTrio.zig");
 pub const ziggurat = @import("rand/ziggurat.zig");
 
+const uuid_hex_table: [256]*const [2:0]u8 = .{
```

Comment:
You can use `std.fmt.bufPrint()` and `{x:0>2}` to format numbers to 0-padded hex strings.

Reply:
@jedisct1 What's the code generation like for `std.fmt.bufPrint()`? I have a C background, so the lookup table is the most straightforward way I thought to implement this.

Reply:
@jedisct1 Actually, nevermind. I'm going to experiment with it and see for myself. It would certainly be cleaner to use `std.fmt.bufPrint()` if it generates similar code.

Reply:
A lookup table is likely to be faster, but in the context of UUID generation, it shouldn't make much of a practical difference. Improving the formatter code with a specialized code path for `{x:0<2}` could be a better approach, that would avoid redundancy and benefit to more applications.